src/main/scala/org/apache/spark/sql/streaming/sqs/SqsClient.scala

```diff
@@ -82,7 +82,7 @@ class SqsClient(sourceOptions: SqsSourceOptions,
     sqsFetchMessagesThread,
     0,
     sqsFetchIntervalSeconds,
```

Comment:
please change the variable names in code / tests and documentation accordingly to milliseconds to avoid confusion